examples/go-client/ruleEngine/engine.go

```diff
@@ -0,0 +1,127 @@
+// +build sample_client
+
+package ruleEngine
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"github.com/Comcast/rulio/core"
+	"github.com/Comcast/rulio/cron"
+	"github.com/Comcast/rulio/examples/go-client/configuration"
+	"github.com/Comcast/rulio/examples/go-client/service"
+	"github.com/Comcast/rulio/sys"
+	"os/exec"
+	"strings"
+	"time"
+)
+
+type EnginePoc struct {
```

Comment:
Is there a way to refactor this code to use `sys.System`? Looks like you are duplicating a lot of code.